en/subscription_services_tos.md

```diff
@@ -1,15 +1,17 @@
 # Mozilla Subscription Services | Terms of Service
 
-Version 1.2, effective January 10, 2024
-{: datetime="2024-01-10" }
+Version 1.2, effective February 6, 2024
```

Comment:
- Should we change the version? 
- The date is different from the PN date. Let's stick to the same release date.

Reply:
yes. I aligned the release date to Feb 5 and updated the version number to 2.0 (I think adding Mozilla Monitor is a major version change). 

I agree with @flodolo's changes too. Thank you